doc/development-guidelines/documentation-guide.rst

```diff
@@ -32,3 +32,23 @@ Publishing lsst.io Site
 =======================
 
 This functionality is included in the DevelopPipeline for the Jenkins Shared Library.
+
+Switching to the guide theme
+============================
+
+We need to create a documenteer.toml file using https://documenteer.lsst.io/guides/toml-reference.html as reference material.
+This is so we become compatible with documenteer 1.0 and greater.
+A basic documenteer.toml example is provided below.
+
+.. literalinclude:: ./documenteer.toml.example
+
+You also need to change first line in conf.py to import from the guide config with
+
+.. code:: python
+
+    from documenteer.conf.guide import *
+
+You can find a template conf.py within the https://github.com/lsst-ts/cookiecutter_tssw_csc_doc repository.
```

Comment:
Please extract this to a separate section, as I originally requested, and describe what this file is for.

Reply:
Added a description of the conf.py to the doc folder template section as that is most likely the appropriate section, added a note that the switching to guide theme section will be removed once the doc folder template is updated. I also added several todo comments once the ticket updating the doc template is completed.